README.it_IT.md

```diff
@@ -0,0 +1,83 @@
+
+Il progetto CentOS [ha recentemente annunciato](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio di strategia per CentOS. Laddove precedentemente CentOS esisteva come *downstream* build del suo vendor (riceveva patch e aggiornamenti dopo l'upstream vendor), adesso diventerà una *upstream* build (testing patches e aggiornamenti ricevuti *prima* dell'upstream vendor).
+
+Inoltre, il supporto per CentOS Linux 8 è stato accorciato, dal 31 Maggio 2029 al 31 Dicembre 2021.
+
+
+> **Q:** Quindi come si posiziona Rocky Linux?
+
+Rocky Linux punta ad essere una downstream build come lo è stato CentOS in precedenza, effettuando rilasci dopo l'upstream vendor, non prima.
+
+> **Q:** Quando sarà rilasciato?
+
+Non c'è ancora un ETA per il rilascio.
+
+> **Q:** Qual è la vision per Rocky Linux?
+
+Un'alternativa **solida**, **stabile**, e **trasparente** per gli ambienti di produzione, sviluppata *dalla* community *per* la community.
+
+> **Q:** Chi guida lo sviluppo di Rocky Linux?
+
+Noi tutti, Rocky Linux è un progetto community-driven e lo sarà per sempre. Rocky Linux non sarà venduto o guidato da interessi aziendali.
+
+> **Q:** Come posso partecipare?
+
+Per piacere leggi la sezione [contribuire](#contribuire) in fondo.
+
+## Contatti
+
+| Team                          | Contatto                                  |
+|-------------------------------|-------------------------------------------|
+| Stampa                        | outreach@rockylinux.org                   |
+| Sviluppo                      | development@rockylinux.org                |
+| Infrastruttura                | infrastructure@rockylinux.org             |
+| Sicurezza                     | security@rockylinux.org                   |
+| Web e Branding                | web@rockylinux.org, brand@rockylinux.com  |
+
+
+Per tutte le altre domande: hello@rockylinux.org
+
+## Segui gli aggiornamenti
+
+* [Slack](https://join.slack.com/t/hpcng/shared_invite/zt-k5z04bsh-1uqpaD1NsYVP73vzc3uKdQ)
+* [Forum](https://forums.rockylinux.org/)
+* [GitHub](https://github.com/rocky-linux/)
+* [Twitter](https://twitter.com/rocky_linux)
+* [IRC](https://webchat.freenode.net/?channels=rockylinux)
+* [Reddit](https://www.reddit.com/r/RockyLinux)
+* [Matrix](https://matrix.to/#/+rockylinux:matrix.org)
+
+## Contribuire
+
+Grazie per il tuo interesse a contribuire al progetto.
+
+Se sei uno sviluppatore, architect o ingegnere, o in ogni caso interessato a contribuire il tuo tempo e le tue competenze considera di iscriverti al nostro [Slack](https://join.slack.com/t/hpcng/shared_invite/zt-k5z04bsh-1uqpaD1NsYVP73vzc3uKdQ) ed entrare nei canali che rilevanti per i tuoi interessi.
+
+Se sei interessato a donare o sponsorizzare il progetto, per piacere scrivi a hello@rockylinux.org
+
+**NOTA**: Al momento noi **non** abbiamo alcuna forma di crowdsourcing.
+
+## Ringraziamenti speciali
+
+Vorremmo ringraziare i seguenti gruppi per il loro supporto fin qui sul progetto:
```

Comment:
I would rephrase it as `per il loro supporto fin qui mostrato nel progetto`.

Reply:
yes, that needs better phrasing. `per il supporto al progetto fin qui mostrato` sounds better to me, while incorporating your suggestion.